### ISSUE TITLE
Datagrid action overflow divs remain open when using Angular's RouteReuseStrategy

### ISSUE THREAD

**Issue by boldertech** [vmware/clarity#2115]
**Select one ...**  (check one with "x")
```
[X] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior
Datagrid action overflow buttons maintain expected behavior when using a route reuse strategy:
https://angular.io/api/router/RouteReuseStrategy

### Actual behavior
Datagrid action overflow buttons leave the div open and they appear disconnected from the button.

### Reproduction of behavior
https://clarity-light-theme-v11-w8id9t.stackblitz.io

### Environment details

* **Angular version:** 4.x.x
5.2.1

* **Clarity version:**
0.11.10

* **OS and version:** 
Windows 10/ OSX 10.13.3

* **Browser:** 
Chrome & FF

![image](https://user-images.githubusercontent.com/18277297/38043032-257395ec-3273-11e8-9fc0-46ac46806c93.png)

**Comment by boldertech** [comment]
vDigging into it deeper, I think the issue might be in here:
https://github.com/vmware/clarity/blob/v0.10/src/clarity-angular/popover/common/popover-old.directive.ts

I'm still trying to learn Angular, but these two fields might be static:
let openCount: number = 0;
const waiting: Array<() => void> = [];  // pending create functions

So I'm guessing that they are getting shared between the two instances of Datagrid and/or possibly because the ngOnDestroy() is no longer getting called.

I'm not sure why the click on the Nav link is not closing a currently popover, maybe it does. I've also added a router event listener to try and manually close the popovers, but that hasn't worked either:  
https://clarity-light-theme-v11-uxhaug.stackblitz.io/fruits

I haven't figured out how to debug the clarity-angular module yet. If you have any advice on how to get debugging working, I'd appreciate it.

Thanks!

**Comment by boldertech** [comment]
Adding the following to DatagridActionOverflow seems to fix the issue
`
[clrPopoverOldOptions]="{allowMultipleOpen: true}"
`

```
@Component({
    selector: "clr-dg-action-overflow",
    template: `
        <button (click)="toggle($event)" class="datagrid-action-toggle" #anchor>
            <clr-icon shape="ellipsis-vertical"></clr-icon>
        </button>
        <ng-template [(clrPopoverOld)]="open" [clrPopoverOldAnchor]="anchor" [clrPopoverOldAnchorPoint]="anchorPoint"
                     [clrPopoverOldPopoverPoint]="popoverPoint" [clrPopoverOldOptions]="{allowMultipleOpen: true}">
            <div #menu class="datagrid-action-overflow" (clrOutsideClick)="close($event)" [clrStrict]="true">
                <ng-content></ng-content>
            </div>
        </ng-template>
    `
})
```

**Comment by boldertech** [comment]
Ok, I found a work around for it by not using clr-dg-action-overflow and instead creating a custom column using the same styles and using a signpost.
https://clarity-light-theme-v11-tegt8a.stackblitz.io

Hopefully I have time to try and put this into a pull request at some point to fix it, but no time now.

**Comment by gnomeontherun** [comment]
This behavior seems to be resolved by #2923 so I'm going to mark this as fixed.
